### PR TITLE
cli: Support --minify --reactImportSource and place babel plugin before others

### PR DESCRIPTION
**Pull request by jayphelps**
Will discuss with Kai Monday.

### CODE REVIEW

File: packages/cli/src/commands/deploy.ts [hunk 1]
```diff
@@ -279,6 +231,91 @@ export default class Deploy extends Command {
...
+        include: '**',
+        compact: false,
+      }),
+      rollupNodeResolve({
+        browser: true,
+        preferBuiltins: false,
+        extensions: SUPPORTED_EXTENSIONS,
+      }),
+      rollupCommonJs({
+        dynamicRequireTargets: ['node_modules/enquire.js/**/*.js'],
```
Comment (kaicataldo) [inline packages/cli/src/commands/deploy.ts:261]: Do we not need `SUPPORTED_EXTENSIONS` here after all? If so, should we remove it from the config for analysis thunks rollup run as well?

Comment (jayphelps) [reply packages/cli/src/commands/deploy.ts:261]: Good catch. it's not needed for the turbo stuff, but I didn't intentionally remove it. Adding it back as I don't see any harm at the moment.

**Review by kaicataldo — DISMISSED** [review]
LGTM

File: packages/cli/src/commands/deploy.ts [hunk 2]
```diff
@@ -279,6 +231,92 @@ export default class Deploy extends Command {
     }
   }
 
+  async rollupOutsmartlyConfigFile(configPath: string): Promise<{ chunk: OutputChunk; bundle: RollupBuild }> {
+    const { flags } = this;
+    const plugins = [
+      rollupBabel({
```
Comment (kaicataldo) [inline packages/cli/src/commands/deploy.ts:237]: I spent some time trying to get this to work and also ran into issues when the babel plugin didn't run first. Seems like we should maybe also change the analysis artifacts rollup run config to match this as well, though I don't think we have to do it in this PR.

Looks like [this is the reason they say to put the commonjs plugin before the babel one](https://github.com/rollup/rollup-plugin-commonjs/issues/146), but if we're not being bitten by it then that seems fine. I'm mostly just confused why it seems to work for some cases but not others...

**Review by kaicataldo — APPROVED** [review]

**Comment by jayphelps** [comment]
Released as 0.0.79

**Comment by jayphelps** [comment]
Scratch that. I forgot to build the CLI so I need to publish as 0.0.80

**Pull request merged**